minion/plugins/snmp-plugin/src/main/java/org/opennms/horizon/minion/snmp/SnmpMonitor.java

```diff
@@ -126,19 +125,19 @@ public CompletableFuture<ServiceMonitorResponse> poll(MonitoredService svc, Any
                 throw new IllegalArgumentException("config must be an SnmpRequest; type-url=" + config.getTypeUrl());
             }
 
-            SnmpMonitorRequest snmpMonitorRequest = config.unpack(SnmpMonitorRequest.class);
+            snmpMonitorRequest = config.unpack(SnmpMonitorRequest.class);
```

Comment:
```suggestion
final SnmpMonitorRequest snmpMonitorRequest = config.unpack(SnmpMonitorRequest.class);
```
then you don't need another final SnmpMonitorRequest finalSnmpMonitorRequest

Reply:
then snmpMonitorRequest is out of scope in exception block

Reply:
you don't need to initialize with null at the start

Reply:
Discussed